i3lock.1

```diff
@@ -74,6 +76,15 @@ verified or whether it is wrong).
 .BI \-i\  path \fR,\ \fB\-\-image= path
 Display the given PNG image instead of a blank screen.
 
+.TP
+.BI \-r\  format \fR,\ \fB\-\-raw= format
+Read the image given by \-\-image as a raw image instead of PNG. The argument is the image's format
+as <width>x<height>:<pixfmt>. The supported pixel formats are 'native' and 'rgb'.
+The "rgb" pixel format expects a pixel to be three bytes; red, green, and blue.
+The "native" pixel format expects a pixel to be ARGB, as a 32-bit (4-byte) integer in
+the machine's native endianness. The alpha is ignored.
```

Comment:
Why does it need to be called "alpha" if it is ignored?

I am asking, because cairo uses pre-multiplied alpha: Red with 50% transparency has value 0x7f in the red component, not 0xff. If you just do not mention it as alpha, this problem "goes away".

Here is the doc for `CAIRO_FORMAT_RGB24`, for "wording inspiration":

> each pixel is a 32-bit quantity, with the upper 8 bits unused. Red, Green, and Blue are stored in the remaining 24 bits in that order.